### ISSUE TITLE
Saving Project Properties Duplicates Groups - VS2015 U3

### ISSUE THREAD

**Issue by dan-def** [Microsoft/TypeScript#11345]
**TypeScript Version:**  2.0.3

Making changes to the project settings in the TypeScript Build tab in Visual Studio 2015 Update 3 results in duplicate `<PropertyGroup>` entries in the csproj file which are then ignored.

I modified the "Combine JavaScript output into this file" option and saved the changes. Another `<PropertyGroup>` was added to my csproj file which included the inputted value (as well as some other empty options). I then modified the path and another `<PropertyGroup>` was added which included my modified path in addition to the `<PropertyGroup>` added in the previous save.

I then rebuilt the project and the option was completely ignored and the files were build separately as they were previously. In order to fix the issue I had to manually move the `<TypeScriptOutFile>` entry into the original property group that was created when I first started the project and delete the others added during this exercise.

**Comment by yuit** [comment]
@dan-def Sorry for delay here. I know it has been a while but just to get a bit more information here - do you happen to use NuGet pkg with this TypeScript project?

**Comment by dan-def** [comment]
@yuit yes, we are using the the Microsoft.TypeScript.MSBuild NuGet pkg.

**Comment by paulvanbrenk** [comment]
I have a fix which should be part of TypeScript 2.3 and Visual Studio 2017 Update 3. 

As a workaround. If you add
```
 <Import Project="Microsoft.TypeScript.targets" Condition="Exists('Microsoft.TypeScript.targets')" />
 ```
before the line where the `Microsoft.TypeScript.MSBuild.targets` are imported changing the project properties should work as expected.
